### PR TITLE
Add OpenSBI PPI

### PR DESCRIPTION
**Pull request by JohnAZoidberg**

### CODE REVIEW

File: Platform/SiFive/U5SeriesPkg/FreedomU540HiFiveUnleashedBoard/U540.dsc [hunk 1]
```diff
@@ -385,6 +385,8 @@
     NULL|MdeModulePkg/Library/LzmaCustomDecompressLib/LzmaCustomDecompressLib.inf
   }
 
+  RiscVPkg/Universal/OpenSbi/Pei/OpenSbiPeim.inf
```
Comment (changab) [inline Platform/SiFive/U5SeriesPkg/FreedomU540HiFiveUnleashedBoard/U540.dsc]: Could you do me a favor to rename it to **RiscVOpensbi**\Pei\RiscVOpensbiPeim.inf
Make it consistent with the naming of RiscVOpensbiLib

File: Platform/SiFive/U5SeriesPkg/FreedomU500VC707Board/U500.dsc [hunk 2]
```diff
@@ -391,6 +391,8 @@
       PcdLib|MdePkg/Library/PeiPcdLib/PeiPcdLib.inf
   }
 
+  RiscVPkg/Universal/OpenSbi/Pei/OpenSbiPeim.inf
```
Comment (changab) [inline Platform/SiFive/U5SeriesPkg/FreedomU500VC707Board/U500.dsc]: Could you do me a favor to rename it to RiscVOpensbi\Pei\RiscVOpensbiPeim.inf
Make it consistent with the naming of RiscVOpensbiLib

File: Platform/SiFive/U5SeriesPkg/FreedomU540HiFiveUnleashedBoard/U540.dsc (hunk 1 above)
Comment (JohnAZoidberg) [reply Platform/SiFive/U5SeriesPkg/FreedomU540HiFiveUnleashedBoard/U540.dsc]: Done

File: Platform/SiFive/U5SeriesPkg/FreedomU500VC707Board/U500.dsc (hunk 2 above)
Comment (JohnAZoidberg) [reply Platform/SiFive/U5SeriesPkg/FreedomU500VC707Board/U500.dsc]: Done

**Comment by JohnAZoidberg** [comment]
If we create the PPI in SEC, this is not necessary anymore.
See: changab/edk2-staging-riscv#7

**Pull request closed without merging**